mongodbatlas/cloud_provider_snapshot_backup_policies_test.go

```diff
@@ -304,7 +304,7 @@ func TestCloudProviderSnapshotBackupPolicies_Delete(t *testing.T) {
 	groupID := "5b6212af90dc76637950a2c6"
 	clusterName := "myCluster"
 
-	path := fmt.Sprintf("/groups/%s/clusters/%s/backup/schedule", groupID, clusterName)
```

Comment:
This test was merged yesterday here https://github.com/mongodb/go-client-mongodb-atlas/pull/210